core/domain/stats_services.py

```diff
@@ -77,7 +91,12 @@ def get_versions_for_exploration_stats(exploration_id):
 def get_exploration_stats(exploration_id, exploration_version):
     """Returns a dict with state statistics for the given exploration id.
 
-    Note that exploration_version should be a string.
+    Args:
+        exploration_id: str. The exploration ID.
+        exploration_version: str. The version of the exploration.
```

Comment:
I logged exploration_version. When I expected string I get 'all' as version. And an integer when I expect int. I couldn't think of any better way to verify it. Would be happy to learn of any techniques to verify this.